packages/uma/src/routes/Policy.ts

```diff
@@ -0,0 +1,75 @@
+import { getLoggerFor } from "@solid/community-server";
+import { UCRulesStorage } from "@solidlab/ucp";
+import { HttpHandlerContext, HttpHandlerResponse, HttpHandler, HttpHandlerRequest } from "../util/http/models/HttpHandler";
+import { Quad, Writer, DataFactory } from "n3";
+
+// Need this for the 
+const { namedNode } = DataFactory;
+
+// ODRL implementation, notice that we currently give every policy where client is
+// mentioned with 'permission', 'prohibition' and 'duty'.
+const ordlAssigner = namedNode('http://www.w3.org/ns/odrl/2/assigner');
+const relations = ['permission', 'prohibition', 'duty']
+
+/**
+ * Endpoint to handle policies, this implementation gives all policies that have the
+ * client as assigner.
+ */
+export class PolicyRequestHandler extends HttpHandler {
+
+    protected readonly logger = getLoggerFor(this);
+
+    /**
+     * Mock function to extract client from the request, because it is not known yet what
+     * this request will look like in the future.
+     * 
+     * @param request the request with the client 'id' as body
+     * @returns the client id
+     */
+    private getClient(request: HttpHandlerRequest): string {
+        return request.body as string;
+    }
```

Comment:
I think it makes sense for the HTTP request to be a GET request.
That means a body will not be present.

We will likely use an Authorization header at some point (to use Solid-OIDC)
So maybe for now already check the authorization header?

For now, it can be a simple header `Authorization: webID`

We can later look into a different kind of authorization header parses that verifies Solid-OIDC tokens